tests/main/views/test_download_responses.py

```diff
@@ -413,7 +426,10 @@ class TestDownloadBriefResponsesCsv(BaseApplicationTest):
 
     def setup_method(self, method):
         super(TestDownloadBriefResponsesCsv, self).setup_method(method)
-        self.brief = BriefStub(status='closed').single_result_response()
+
+        # CSV download only works with DOS1
+        # TODO: Fix or remove this functionality?
```

Comment:
Buyers can still look at their old briefs. I'd keep these tests, and make it clear that we're using DOS1.

Reply:
👍